PackageExplorer/Converters/PackageIconConverter.cs

```diff
@@ -28,12 +28,18 @@ public class PackageIconConverter : IMultiValueConverter
                     {
                         if (string.Equals(file.Path, iconPath, StringComparison.OrdinalIgnoreCase))
                         {
-                            var image = new BitmapImage();
-                            image.BeginInit();
-                            image.CacheOption = BitmapCacheOption.OnLoad;
-                            image.StreamSource = file.GetStream();
-                            image.EndInit();                            
-                            return image;
+                            // catch potential exceptions during image loading from zip file
+                            // see 1097
+                            try
+                            {
+                                var image = new BitmapImage();
+                                image.BeginInit();
+                                image.CacheOption = BitmapCacheOption.OnLoad;
+                                image.StreamSource = file.GetStream();
```

Comment:
Is this really going to catch it though? It depends on where the exception happens since this by this point it's just opening the stream. I think load happens by the BitmapImage happens later.

Is there a call stack that shows the crash directly?

Reply:
See https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/issues/1097#issuecomment-691667911